tools/check_package_release.py

```diff
@@ -0,0 +1,58 @@
+"""Determine if new package should be release (based on version) and consequently write information to env."""
+import os
+import sys
+
+import requests
+
+VERSION_MATCHSTR = r'\s*__version__\s*=\s*"(\d+\.\d+\.\d+)"'
+RESPONSE_ERROR = 404
+
+
+def _check_no_version_pypi(pypi_endpoint, package_name, package_version):
+    response = requests.get(pypi_endpoint, timeout=10)
+    if response.status_code == RESPONSE_ERROR:
+        # Version doesn't exist on Pypi - do release
+        print(f"Potential release of {package_name} {package_version}")  # noqa: T201
+        return True
+    else:
+        print(f"Skipped: {package_name} {package_version} already exists on PyPI")  # noqa: T201
+        return False
+
+
+def _check_no_dev_version(package_name, package_version):
+    if "dev" not in package_version:
+        return True
+    else:
+        print(f"Skipped: {package_name} {package_version} is still under development")  # noqa: T201
+        return False
+
+
+if __name__ == "__main__":
+    """Check if a package needs to be released"""
+    new_release = "false"
```

Comment:
Why are we processing the `new_release` value with a string instead of a bool.

Reply:
good point - it was taken over from a script in kedro, let me try to change it